### ISSUE TITLE
Homepage Feedback

### ISSUE THREAD

**Issue by GabeIsman** [mediapublic/mediapublic#157]
I've got some comments on the home page design, I should've put them in the PR maybe, but mostly they just occurred to me so here goes:
1. It feels a bit cramped. I think it could benefit from a little more whitespace.
2. I want it to be centered. It's close enough to centered that it feels like it should just be centered. Probably just a personal preference, but it gives me kind of an uneasy feeling.
3. Where's the navigation? Does it only show up on other pages? Is there none? Would be great to get a clearer sense of this.
4. I don't really like the tagline under the 'logo'/site name. It doesn't seem like that's important enough to be on every page. Feels like it belongs over a nice image somewhere in the body of the homepage.
5. The copy is very sparse. It feels like there should be an explanation of what the site is and what you might use it for. It's pretty hard to figure that out from the current copy IMO. 'Find great content' in particular feels very generic.
6. What does the 'pre-populated query' link to? I'm not currently sure where that would go. Which also means that I don't really know what it means in the context of our site.
7. Are the selection of tiles curated by hand? I don't think we've scoped in any kind of curation method.

I'm sure some of this is just my personal preference, so feel free to ignore it if you feel strongly about it, but I thought I would mention it.

**Comment by GabeIsman** [comment]
I'm working on implementing the home page and I'm still struggling with some of these concerns. In particular right now I'm in a fight with the dark blue color. It's very dark. It also really doesn't work layered over an image. Just playing around with this a bit here:

<img width="1436" alt="screenshot 2016-01-12 12 41 46" src="https://cloud.githubusercontent.com/assets/697801/12276338/19ffe4f0-b92a-11e5-9347-0174ff7be00c.png">

**Comment by thequbit** [comment]
I personally don't have an issue with the dark blue.  I think if we go much lighter we get into the territory of "Facebook blue".

I do like what you did with the logout area!

**Comment by nkkl** [comment]
1. Awesome feedback, thanks @GabeIsman for taking the time to write this up! I also love pulling in a hero image! Let's throw our description and call to action (e.g. prepopulated query/search) into that.
2. The whitespace and typography still feels wonky to me, too, but I've been letting things be "good enough" until I can get out a first draft of every undesigned page. I'll keep this in mind as I go back through this design, though.
3. Something something CSS. Tuning the typography and copy lengths should help a bunch with this.
4. Right now the nav is really lightweight, but as we talked about recently, I'm taking another pass at that. I'd like the front page to stay as action-oriented and low cognitive load as possible, so I'm going to argue for not surfacing a lot of navigational structure just yet. In the beginning, the most powerful thing we can do is get you from the home page to an interesting station page as quickly as possible. There will be more to guide you towards as the site gets fleshed out with content, so this is something that should definitely evolve.
5. Fair. What are your thoughts on the latest header design? (which will most likely get some lightweight nav added to it)
6. Absolutely agreed that the copy on it needs another rev. I'll get together with @melodykramer and noodle on that.
7. The goal with the search stuff and prepopulated query was to unstick people and give them an immediate, easy action to take. Tweaking copy should help unstick that - it certainly shouldn't say "pre-populated query", but rather something like "public media stations in Vermont" or "how tos about code" or... something. Will get @melodykramer to help with this too.
8. These are going to be hand curated to start with. I believe Melody stepped up on that one. :)

**Comment by GabeIsman** [comment]
1. Thanks for the thoughtful response @nkkl! Will do.
2. Totally understand that. I'm doing exactly the same thing on my end :).
3. I like the current header! As I mentioned I worry a little that the color is too dark, but it's a good height I think.
4. Great.
5. I think that's a great idea, the trouble is that I'm not sure our search is going to work well enough to make interesting queries possible. Currently we pretty much only support searching for organizations things by name. We could say something like "Search for your local station" to give them a prompt (or even pre-populate with our best guess for their local station... v2 maybe).
6. OK, I've created a simple file that will contain curated data for this. https://github.com/mediapublic/mediapublic/blob/develop/client/app/index/homepagedata.js. Which instead of all that 'fakes' nonsense should be a list of `{ type: 'organization', id: 'blah-blah-some-id' }`. Not a perfect system but good enough for now. I can help get that formatted correctly @melodykramer.

**Comment by nkkl** [comment]
Good point about search. "Search for your local station" sounds like a good way to set up users for success.

**Comment by nkkl** [comment]
~~@GabeIsman Can you drop the latest screenshot of the version you're working on, for easier reference in this thread? Lots of good stuff there!~~ (nvm I'm silly) (although it would be good to have the latest thing in here)

I'm starting to trawl through Flickr and my other go-to CC image sources to see what options we have for hero images.
